src/App.jsx

```diff
@@ -11,9 +12,18 @@ function App() {
       <Routes>
         <Route path="/articles" element={<Home />} />
         <Route path="/articles/:article_id" element={<ArticlePage />} />
+        <Route
```

Comment:
Good that you've added this, however as you don't have a path for '/', it's resulting in this error when you go to the root of the website 👀